css/types/timing-function.json

```diff
@@ -0,0 +1,223 @@
+{
+  "css": {
+    "types": {
+      "timing-function": {
+        "__compat": {
+          "description": "<code>&lt;timing-function&gt;</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/single-transition-timing-function",
+          "support": {
+            "webview_android": {
+              "version_added": "4"
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "4"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "10.5"
+            },
+            "opera_android": {
+              "version_added": "10"
+            },
+            "safari": {
+              "version_added": "3.1"
+            },
+            "safari_ios": {
+              "version_added": "2"
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "cubic_bezier": {
```

Comment:
Would you mind changing this to a dash instead of an underscore?

Reply:
I can, but I'm not sure that it will be right. I saw that other CSS features in repository have an underscore format. For the <code>description</code> property i used a dash.

Reply:
I think there are some inconsistencies in the data, but generally I think the current practice is to use a dash when it's consistent with the name of the keyword, function, etc. itself (e.g., in this same directory, `basic-shape`) and underscores in place of what would otherwise be a space in a key (e.g., a description like `viewport_lengths_invalid_in_paged_media` in `length.json` or a name like `global_keywords`). Does that help clear things up here?